hazelcast/serialization/compact.py

```diff
@@ -1754,7 +1760,10 @@ def _init(self):
 
         for field in self.fields_list:
             kind = field.kind
-            if FIELD_OPERATIONS[kind].is_var_sized():
+            op = FIELD_OPERATIONS[kind]
```

Comment:
Should we do an index check for this?

Reply:
Added at: https://github.com/hazelcast/hazelcast-python-client/pull/550/commits/49fed9dbfaf3ea39d0103e9834bc486af47d2f4c